src/utils/addresses.ts

```diff
@@ -0,0 +1,52 @@
+import { getAddress } from '@ethersproject/address'
+
+// returns the checksummed address if the address is valid, otherwise returns false
+export function isAddress(value: any): string | false {
+  try {
+    // Alphabetical letters must be made lowercase for getAddress to work.
+    // See documentation here: https://docs.ethers.io/v5/api/utils/address/
+    return getAddress(value.toLowerCase())
+  } catch {
+    return false
+  }
+}
+
+// Shortens an Ethereum address
+export function shortenAddress(address: string): string {
+  const parsed = isAddress(address)
+  if (!parsed) return ''
+  return ellipseAddressAdd0x(parsed)
+}
+
+// Shorten the checksummed version of the input address to have 0x + 4 characters at start and end
+export function shortenAddressStrict(address: string): string {
+  const parsed = isAddress(address)
+  if (!parsed) {
+    throw Error(`Invalid 'address' parameter '${address}'.`)
+  }
+  return ellipseAddressAdd0x(parsed)
+}
+
+/**
+ * Shorten an address and add 0x to the start if missing
+ * @param target
+ * @param charsStart amount of character to shorten (from both ends / in the beginning)
+ * @param charsEnd amount of characters to shorten in the end
+ * @returns formatted string
+ */
+function ellipseAddressAdd0x(address: string, charsStart = 4, charsEnd = 4): string {
```

Comment:
```suggestion
 * @param targetAddress
 * @param charsStart amount of character to shorten (from both ends / in the beginning)
 * @param charsEnd amount of characters to shorten in the end
 * @returns formatted string
 */
function ellipseAddressAdd0x(targetAddress: string, charsStart = 4, charsEnd = 4): string {
```

Reply:
trying to unify the variable name with the docs, as you had in the implementation below